service/raft/raft_address_map.hh

```diff
@@ -61,9 +61,7 @@ class raft_address_map {
         }
     };
 
-    using lru_list_hook = bi::list_base_hook<>;
-
-    class expiring_entry_ptr : public lru_list_hook {
+    class expiring_entry_ptr : public bi::list_base_hook<> {
```

Comment:
Seams to be unrelated. Just a way to reduce line count :)

Reply:
Yes :P

previously `lru_list_hook` was used in two places, now it's used in exactly one place, here, so I thought there's no point for doing the alias.